packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -149,13 +149,19 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
       groups
     });
 
+    const expandBtnArialLabel = expandButtonAriaLabel
+      ? expandButtonAriaLabelUseName
+        ? `${link.name} ${expandButtonAriaLabel}`
```

Comment:
💯 
I'd vote that we change the default case and make it the best possible experience instead of adding another prop.

Reply:
Thanks @cschlechty @jdhuntington 
I removed the extra prop and updated the aria label so that the link name is always prepended, or if the label is not provided, it will default to the link name. See my updated PR description for examples of resulting strings